src/components/dashboard/TeamMembersTable/TeamEditInfo.js

```diff
@@ -0,0 +1,121 @@
+  const [editTeamOrg, { data: dataOrgChange, loading: loadingOrgChange }] = useMutation(UPDATE_TEAM_ORGANIZATION);
+
+  useEffect(() => {
+    if (dataNameChange) {
+      const name = dataNameChange.updateTeamName.name;
+      dispatch({ type: UPDATE_USER_TEAM_NAME, payload: name });
+      toast({
+        position: 'top',
+        render: props => <Toast {...props} description="You have updated your team name" isClosable />,
+      });
+    }
+  }, [dataNameChange, dispatch, toast]);
+
+  useEffect(() => {
+    if (dataOrgChange) {
+      const organization = dataOrgChange.updateTeamOrganization.organization;
+      dispatch({ type: UPDATE_USER_TEAM_ORGANIZATION, payload: organization });
+      toast({
+        position: 'top',
+        render: props => <Toast {...props} description="You have updated your team affiliation" isClosable />,
+      });
+    }
+  }, [dataOrgChange, dispatch, toast]);
+
+  const onChangeTeamName = e => {
+    setNewTeamName(e.target.value);
+  };
+
+  const onChangeTeamOrg = e => {
+    setNewTeamOrg(e.target.value);
+  };
+
+  const mutationUpdateTeam = e => {
+    e.preventDefault();
+    try {
+      if (newTeamName) {
+        editTeamName({ variables: { id: teamId, name: newTeamName } });
+      }
+      if (newTeamOrg) {
+        editTeamOrg({ variables: { id: teamId, organization: newTeamOrg } });
+      }
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  return (
+    <>
+      <Box>
+        <Button
+          color="black"
+          onClick={() => {
+            setOnEditPage(false);
+          }}
+          variant="link"
+          mb={2}
+        >
+          {'< Back'}
+        </Button>
+        <Heading as="h1" size="h1" mb={16}>
+          Edit Team Details
+        </Heading>
+      </Box>
+      <chakra.form w="100%" onSubmit={mutationUpdateTeam}>
+        <VStack spacing="24px" w="100%" h="100%" pl={10} align="flex-start">
+          <Heading alignSelf="flex-start" size="h2" as="h2" mb={1}>
+            Team Details
+          </Heading>
+          <Box w="100%">
+            <FormControl id="teamName">
+              <FormLabel textTransform="uppercase" as="h4" size="subtitle" color="gray.500" mb="8px">
+                Team Name
+              </FormLabel>
+              <Input name="teamName" placeholder={teamName} value={newTeamName} onChange={onChangeTeamName} w="40%" />
+            </FormControl>
+          </Box>
+          <Box w="100%">
+            <FormControl id="teamOrg">
+              <FormLabel textTransform="uppercase" as="h4" size="subtitle" color="gray.500" mb="8px">
+                Team Affiliation
+              </FormLabel>
+              <Input name="teamOrg" w="40%" placeholder={teamOrg} value={newTeamOrg} onChange={onChangeTeamOrg} />
+            </FormControl>
+          </Box>
+          <FormErrorMessage>Something went wrong, please try again later</FormErrorMessage>
```

Comment:
1. THis should be nested in a `FormControl`
2. the form control needs to be controlled by the `isInvalid` prop (you can try `isInvalid={<yourMutationErrorResult>}`

> If you render FormErrorMessage and isInvalid is false or undefined, FormErrorMessage won't be visible. The only way to make it visible is by passing isInvalid and setting it to true.
https://chakra-ui.com/docs/form/form-control